Assets/Input_Test/Scripts/02_Input/ClickOnDropdown.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class ClickOnDropdown : MonoBehaviour {
```

Comment:
Please run format.sh or format.ps1 found in the root of the repo. Otherwise the next commit running the format tools will have to recommit a reformat of all the files here.